server/config/options.go

```diff
@@ -67,9 +68,31 @@ var DefaultConfig = Config{
 		StreamBatch:    100,
 		StreamBuffer:   200,
 	},
+	Search: SearchConfig{
+		Host:         "0.0.0.0",
+		Port:         8108,
+		ReadEnabled:  true,
+		WriteEnabled: true,
+		AuthKey:      "Hu52dwsas2AdxdE",
```

Comment:
This is just for local dev and it is public key provided by TS to play-around. Once we have a way to generate a key then it will be removed from here.